src/lib/components/segmentedControl/index.test.tsx

```diff
@@ -1,4 +1,4 @@
-import { fireEvent, render } from '@testing-library/react';
+import { render } from '../../util/testUtils';
```

Comment:
This is a team style preference, we tried to use absolute paths whenever possible to keep things clear and nice:

```suggestion
import { render } from 'util/testUtils';
```

Reply:
Doesn't seem to work on dirty swan 😢 I need to check but maybe we're missing some alias configuration

Reply:
Ohhh yeah, we probably need to update the TS config file. Ok all good on this PR then.